Sources/AWSLambdaRuntimeCore/LambdaConfiguration.swift

```diff
@@ -17,42 +17,23 @@ import Logging
 import NIOCore
 
 internal struct LambdaConfiguration: CustomStringConvertible {
-    let general: General
-    let lifecycle: Lifecycle
-    let runtimeEngine: RuntimeEngine
-
-    init() {
-        self.init(general: .init(), lifecycle: .init(), runtimeEngine: .init())
-    }
-
-    init(general: General? = nil, lifecycle: Lifecycle? = nil, runtimeEngine: RuntimeEngine? = nil) {
-        self.general = general ?? General()
-        self.lifecycle = lifecycle ?? Lifecycle()
-        self.runtimeEngine = runtimeEngine ?? RuntimeEngine()
-    }
+    var general: General = .init()
+    var lifecycle: Lifecycle = .init()
+    var runtimeEngine: RuntimeEngine = .init()
```

Comment:
This is probably unnecessary if you don't like it but I updated this to be easier to read. You don't necessarily need the init functions you had before if these are just structs. Updating to vars for these internal structs still keeps the same ability to initialize these structs with the default values you had defined in your previous init functions.

Let me know if you don't want this!